README.md

```diff
@@ -980,6 +980,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Mujahid Ali](https://github.com/MujahidAli954)
   - [Mukhtar Abdulfatai](https://github.com/DevMukhtarr)
```

Comment:
This line [981] is gonna be the correct order If you put your name in it to satisfy the alphabetical order requirement to get your PR merged